src/routes/orders.js

```diff
@@ -1,95 +1,90 @@
 const express = require("express");
+const bodyParser = require("body-parser");
+let jsonParser = bodyParser.json();
```

Comment:
Could you firstly group all imports first and then execute logic with `express.Router()` and vise versa?